docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -1250,6 +1250,64 @@ When I wait until scroll is finished
 Then page is scrolled to element located by `id(footerElement)`
 ----
 
+==== Script Steps
+
+WARNING: These steps are deprecated and will be removed in VIVIDUS 0.7.0.
+
+The Script Steps verify the presence of JavaScript files in the HTML source code of an active page.
+They focus on validating specific JavaScript files using attributes like FileName, Text, and TextPart.
+
+=== Validate JavaScript files is present in HTML source code
+
+These step primarily focus on checking for the existence of certain JavaScript files based on FileName attributes.
+
+[source,gherkin]
+----
+Then a javascript file with the name '$jsFileName' is included in the source code
+----
+
+* `$jsFileName` - The JavaScript FileName. In order to save a result return statement should be used.
+
+.Checks the script for the presence of a document by name
+[source,gherkin]
+----
+Then a javascript file with the name 'app' is included in the source code
```

Comment:
please, make examples for more close to real, obviously there couldn't be JS files with name `app`